scripts/ScriptUtilities.psm1

```diff
@@ -61,15 +61,10 @@ function New-NuGetPackageFromNuspecFile($configuration, $project, $version, $fra
 }
 
 function New-NuGetPackages($configuration, $projects, $frameworks) {
-    $versionPrefix, $versionSuffix = & $PSScriptRoot\Get-VersionConstants.ps1
-    if ($versionSuffix)
-    {
-        $version = "$versionPrefix-$versionSuffix"
-    } else {
-        $version = $versionPrefix
-    }
-
+    dotnet tool install --global nbgv --version 3.3.37
+    $version = nbgv get-version -p src -v Version
```

Comment:
>-p [](start = 32, length = 2)

If there are long-form versions of these command line parameters `-p` and `-v`, use them to improve the readability of the script.

Reply:
There is another script New-VersionConstantsFile.ps1 that you should be able to remove.

---
In reply to: [521542194](https://github.com/microsoft/sarif-visualstudio-extension/pull/267#discussion_r521542194) [](ancestors = 521542194)